netmanager/src/views/pages/Map/OverlayMap.js

```diff
@@ -441,7 +467,7 @@ export const OverlayMap = ({
           el.className = `marker ${
             seconds >= MAX_OFFLINE_DURATION ? "marker-grey" : markerClass
           }`;
-          el.innerText = (pollutantValue && pollutantValue.toFixed(0)) || "n/a";
+          el.innerText = (pollutantValue && pollutantValue.toFixed(0)) || "N/A";
```

Comment:
I was wondering....for cases where we do not have measurements, why don't we just remove them from the map? Does this N/A serve any purpose? What are the alternatives? @FaithDaka @NoahNsimbe